operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md

```diff
@@ -30,240 +42,288 @@ Scenarios where this procedure is applicable:
 
-                    ```text
-                    4    a4:bf:01:65:68:54    dynamic        1/1/48
-                    ```
+    1. Query Kea for the BMC MAC address to determine its current IP address.
 
-                * Aruba Management Switch
+        ```bash
+        BMC_IP=$(curl -sk -H "Authorization: Bearer ${TOKEN}" -X POST -H "Content-Type: application/json" \
+                     -d '{ "command": "lease4-get-all", "service": [ "dhcp4" ] }' \
+                     https://api-gw-service-nmn.local/apis/dhcp-kea |
+                     jq --arg BMC_MAC "${BMC_MAC}" \
+                         '.[].arguments.leases[] | select(."hw-address" == $BMC_MAC)."ip-address"' -r)
+        echo ${BMC_IP}
+        ```
 
-                    ```console
-                    show mac-address-table | include 1/1/48
-                    ```
+        Example output:
 
-                    Example output:
+        ```text
+        10.254.1.26
+        ```
 
-                    ```text
-                    a4:bf:01:65:68:54    4        dynamic                   1/1/48
-                    ```
+        > ***Troubleshooting***
+        > If the MAC addresses of the BMC are not present in Kea, then check for the following items:
+        >
+        > 1. Verify that the BMC is powered up and has an active connection to the network.
+        > 1. Verify that the BMC is set to DHCP instead of a static IP address.
 
-    1. (`ncn-mw#`) **Skip if adding `ncn-m001`:** Set the `BMC_MAC` environment variable to the BMC MAC address.
+    1. (`ncn-mw#`) Ping the BMC to see if it is reachable.
 
         ```bash
-        BMC_MAC=a4:bf:01:65:68:54
+        ping "${BMC_IP}"
         ```
 
-    1. **Skip if adding `ncn-m001`:** Determine the current IP address of the NCN BMC.
+1. (`ncn-mw#`) **Perform this step if adding `ncn-m001`, otherwise skip:** Set the `BMC_IP` environment variable to the current IP address or hostname of the BMC. This is not the allocated HMN address for the BMC of `ncn-m001`.
 
-        1. (`ncn-mw#`) Query Kea for the BMC MAC address to determine its current IP address.
+    ```bash
+    BMC_IP=10.0.0.10
+    ```
 
-            ```bash
-            BMC_IP=$(curl -sk -H "Authorization: Bearer ${TOKEN}" -X POST -H "Content-Type: application/json" \
-                         -d '{ "command": "lease4-get-all", "service": [ "dhcp4" ] }' \
-                         https://api-gw-service-nmn.local/apis/dhcp-kea |
-                         jq --arg BMC_MAC "${BMC_MAC}" \
-                             '.[].arguments.leases[] | select(."hw-address" == $BMC_MAC)."ip-address"' -r)
-            echo ${BMC_IP}
-            ```
+#### Saving/Reloading
 
-            Example output:
+1. (`ncn-mw#`) Save the currently collected values to a loadable script for reloading.
 
-            ```text
-            10.254.1.26
-            ```
+    ```bash
+    cat << EOF > ${NODE}-env-vars.sh
```

Comment:
Going to update this to make the file in either `/metal/recovery`, the s3fs_cache, or the CSM directory in /usr or /opt.